### ISSUE TITLE
Is able to show binding tips after using objed-mode-map bindings like which-key?

### ISSUE THREAD

**Issue by c02y** [clemera/objed#34]
After using `org-mode-map` bindings such as activating a sentence/identifier/object, is able to show the tips of available bindings (the `objed-map` parts in README) automatically like `which-key`, since there are a lot of them.

It seems I can use `?` to get the tips window, but if it can displayed automatically, that is better.

**Comment by clemera** [comment]
There is a user option `objed-auto-wk-top-level-p` you can set. I auto shows the which key help after which-key-idle-delay when activating objed manually. The filtering of actions by object type is a nice idea!

**Comment by c02y** [comment]
I set `objed-auto-wk-top-level` to `t`, but when I activated a block such as string in `""`, or block of code inside "{}", it doesn't not show available keys automatically. But sometimes it show it, such as when I execute `objed-activate`, it will display it.

**Comment by clemera** [comment]
It only shows them on initial activation.  Would you like to see available keys after every command? Or only after you activate certain objects? I think it would be a bit annoying but maybe it's nice to have at the beginning.

**Comment by c02y** [comment]
1. what should I do to activate identifier/object/block/expression, it should show the available bindings
2. what can I do after I activated identifier/object/block/expression, it should show the available bindings

**Comment by c02y** [comment]
This is the configuration I'm currently using:
~~~
(objed-mode)
(setcdr objed-mode-map nil)                     ;; disable mode map in package
(setq objed-auto-wk-top-level t)
(bind-keys* :map objed-mode-map
                        :prefix-map my-objed-prefix
                        :prefix "C-c t"
                        ("t" . hydra-text-motions/body)
                        ("SPC" . objed-activate)
                        ("<left>" . objed--backward-word)
                        ("<right>" . objed--forward-word)
                        ("p" . objed-prev-identifier)
                        ("n" . objed-identifier-object)
                        ("[" . objed-beg-of-object-at-point)
                        ("]" . objed-end-of-object-at-point)
                        ("(" . objed-until-beg-of-object-at-point)
                        (")" . objed-until-end-of-object-at-point)
                        ("," . objed-first-identifier)
                        ("." . objed-last-identifier))
~~~

And I just found that there are other maps such as objed-object-map, objed-map, objed-op-map, objed--indent-map which all provide a lot of bindings, honestly this is too complicated, there are 164 `define-key` bindings in objed.el, it is really necessary to define 164 key bindings in one package?

And another thing is, I still don't know how to exactly use objed.

**Comment by clemera** [comment]
Objed provides many features, as the package provides basically another Editor I don't think that this are too many keybindings. It's still evolving but at some point I will write a tutorial (see #30). I guess you have to wait for that.
